doc/protocols/scion-header.rst

```diff
@@ -408,3 +408,459 @@ Because of its special structure, no PathMeta header is needed. There is only a
+If the verification fails, the packet is dropped.
+
+How to only allow EPIC-HP traffic on a hidden path (and not standard 
+SCION packets) is described in the `Configuration`_ section.
+
+.. -------------------------------------------------------------------
+
+Path Type: EPIC-SAPV
+====================
+The Path Type EPIC-SAPV (EPIC Source Authentication and Path 
+Validation) contains the following parts:
+
+   - An 8-byte Packet Timestamp (same as for EPIC-HP).
+   - A slightly modified SCION header.
+   - A 16-byte *DVF* (Destination Validation Field).
+
+::
+
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                        PacketTimestamp                        |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                          PathMetaHdr                          |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              DVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+SCION Header Modifications
+--------------------------
+EPIC-SAPV contains the standard SCION header with the following 
+adaptations:
+
+   - Two reserved bits of the Meta Header are used to indicate the 
+     EPIC Version (EV).
+   - The size of the MAC (six bytes in standard SCION) inside the 
+     Hop Fields is reduced to two bytes, the four bytes of freed 
+     space are used for the Hop Validation Field (HVF). 
+
+Path Meta Header
+^^^^^^^^^^^^^^^^
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    | C |  CurrHF   |R| RSV |EV |  Seg0Len  |  Seg1Len  |  Seg2Len  |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ 
+
+EPIC Version (EV):
+   - **EV = 0:** *unused*
+   - **EV = 1:** *unused (in the future this may be used for EPIC-HP 
+     with improved path authorization on every AS on the path, which 
+     corresponds to EPIC Level 1 in the paper.)*
```

Comment:
What paper? You need to add that as a reference if you refer to it. I'd rather you would include the relevant bits here such that this part is as stand-alone as possible.

Reply:
Removed the hint to the EPIC paper.